src/fs-utils.js

```diff
@@ -23,6 +23,13 @@ export function findParentFolder(startDir, parentDirName) {
   return currentDir;
 }
 
+export function verifyHooksFolder(desiredHooksPath) {
+  if(!fs.existsSync(desiredHooksPath)) {
+    console.log('Creating hooks directory in .git');
+    fs.mkdir(desiredHooksPath);
```

Comment:
You are using a mix of sync and async methods here, `fs.mkdir` is async. Please be consistent, and return a promise if you go with the async approach. (I think synchronous is fine here). See: https://nodejs.org/api/fs.html#fs_fs_mkdir_path_mode_callback